test4.js

```diff
@@ -0,0 +1,12 @@
+var foo = '' ;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=29725)
```suggestion
var foo = "" ;
```